openfl/protocols/director.proto

```diff
@@ -7,142 +7,132 @@ import "google/protobuf/duration.proto";
 import "federation.proto";
 
 
-message RequestHeader {
-    string sender = 1;
-}
+message GetDatasetInfoRequest {}
 
 // Envoy Messages
 
 message NodeInfo {
-    string name = 1;
-    string adress = 2;
-    bool cuda_available = 3;
-    uint32 memory_size = 4;
+  string name = 1;
+  string adress = 2;
+  bool cuda_available = 3;
+  uint32 memory_size = 4;
```

Comment:
Doesn't it reduce interface readability?

Reply:
https://developers.google.com/protocol-buffers/docs/style?hl=ru